config/routes.rb

```diff
@@ -185,7 +185,9 @@
   delete '/service_bindings/:guid', to: 'service_bindings#destroy'
 
   # service_credential_bindings
-  get '/service_credential_bindings/:guid', to: 'service_credential_bindings#show'
+  resources :service_credential_bindings,
```

Comment:
Would be nice to show this in the VAT call tomorrow just to get an idea why it is not done this way for the other APIs